javav2/example_code/rekognition/src/main/java/com/example/rekognition/VideoDetect.java

```diff
@@ -0,0 +1,286 @@
+    public static void getLabelJob(RekognitionClient rekClient,
+                                   SqsClient sqs,
+                                   String queueUrl) {
+
+        List<Message> messages=null;
+        ReceiveMessageRequest messageRequest = ReceiveMessageRequest.builder()
+                .queueUrl(queueUrl)
+                .build();
+
+        try {
+            messages = sqs.receiveMessage(messageRequest).messages();
+
+            if (!messages.isEmpty()) {
+                //Loop through messages received.
+                for (Message message: messages) {
+                    String notification = message.body();
+
+                    // Get status and job id from notification.
+                    ObjectMapper mapper = new ObjectMapper();
+                    JsonNode jsonMessageTree = mapper.readTree(notification);
+                    JsonNode messageBodyText = jsonMessageTree.get("Message");
+                    ObjectMapper operationResultMapper = new ObjectMapper();
+                    JsonNode jsonResultTree = operationResultMapper.readTree(messageBodyText.textValue());
+                    JsonNode operationJobId = jsonResultTree.get("JobId");
+                    JsonNode operationStatus = jsonResultTree.get("Status");
+                    System.out.println("Job found in JSON is " + operationJobId);
+                    // Found job. Get the results and display.
+
+                    DeleteMessageRequest deleteMessageRequest = DeleteMessageRequest.builder()
+                            .queueUrl(queueUrl)
+                            .build();
+
+                    String jobId = operationJobId.textValue();
+
+                    if (startJobId.compareTo(jobId)==0) {
+
+                        System.out.println("Job id: " + operationJobId );
+                        System.out.println("Status : " + operationStatus.toString());
+
+                        if (operationStatus.asText().equals("SUCCEEDED"))
+                              GetResultsLabels(rekClient);
+                        else
+                            System.out.println("Video analysis failed");
+
+                        sqs.deleteMessage(deleteMessageRequest);
+                    }
+
+                    else{
+                        System.out.println("Job received was not job " +  startJobId);
+
+                        // Delete unknown message. Consider moving message to dead letter queue
+                        sqs.deleteMessage(deleteMessageRequest);
+                    }
+                }
+            }
+
+        } catch(RekognitionException e) {
+            e.getMessage();
+            System.exit(1);
+        } catch (JsonMappingException e) {
+            e.printStackTrace();
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    //Gets the results of labels detection by calling GetLabelDetection. Label

```

Comment:
Displays information about labels as detected by a call to GetLabelDetection. I would omit the second sentence as that belongs as a prerequisite to GetLabelDetection.